data/liquid/2023/2023-05-30-2c.toml

```diff
@@ -0,0 +1,55 @@
+id = "2023-05-30-2c"
+protocol.version = "0.2.0"
```

Comment:
Is line
`protocol.name = "yeast-uniformity"`
missing?

Reply:
yes, I've added it in today's dataset